### ISSUE TITLE
Test and release followers in dapp

### ISSUE THREAD

**Issue by michaelsena** [3box/3box-dapp#474]

**Comment by oznekenzo** [comment]
• Follow user from another tab, following section in my profile is updated
• Open several tabs from one window with Following count at 0, follow each user in those separate tabs and you still get "Following is public" which means public api is still returning 0 Following

**Comment by oed** [comment]
Are you reloading? How long does it take for it to turn from 0 to something more?

**Comment by oznekenzo** [comment]
I'll test now

**Comment by oznekenzo** [comment]
• Inconsistencies between Following data via public api and via instance was probably the biggest issue we were seeing, now seems to have been fixed (!!)

**Comment by oznekenzo** [comment]
• Still seeing `"WARNING! Head hash didn't match the contents`

**Comment by oed** [comment]
Yeah, still seems like something is wrong in orbitdb. The issue is being tracked here: https://github.com/orbitdb/orbit-db/issues/583

**Comment by oznekenzo** [comment]
- [ ] handle flicker on follow update

**Comment by oznekenzo** [comment]
Not seeing the "Following is public" (following list === 0) on multiple tabs on a newish account, seems that its only with an old account / account with tons of usage that Im seeing this.  

Old accounts / accounts with tons of usage start to return followingList numbers that jump in length

Im going to test now whether new accounts start to get buggy over time, or whether its only old accounts / accounts that have gone through some older state that see the jumpiness in followingList lengths.

**Comment by oznekenzo** [comment]
I've gotten an inconsistency in data between that returned from public `getThread` & instance `getPosts` and it's not resolving despite reloads and closing of all tabs :(

**Comment by oznekenzo** [comment]
Actually changing the value, though, was able to make consistent both states...

**Comment by oznekenzo** [comment]
The inconsistency above was after some really heavy stress testing.

At this point my verdict is: it mostly works.  I think for 98% of users and use cases, this feature is going to work without error or inconsistency.
We can seek to improve its consistency incrementally over time, especially with orbitdb expected to work on some of the issues we're seeing.
@michaelsena it would be great if you could test Following for like an hour (I can jump on) with new and old accounts, first in a light and normal-usage way to see if this is good enough to push for now, and then more aggressively again to see if you're getting some of the issues I've seen.
